bin/rmdir/rmdir.c

```diff
@@ -44,7 +44,7 @@ static int vflag;
 int
 main(int argc, char *argv[])
 {
-	int ch, errors;
+	int ch, exit_code;
```

Comment:
I think all these changes can be backed out. There's no need to change this variable.